### PR TITLE
Link to InMemoryFileProvider

### PR DESCRIPTION
**Pull request by dazinator**
As per @pranavkm's comment on #221 - adding a link to my contrib project.
Made it a table in case there are other community contributions.

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @Eilon

File: README.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+| Build System  | Status |
+| :-----------: | :-----  |
+| AppVeyor  | [![AppVeyor](https://ci.appveyor.com/api/projects/status/8pryt261xf7q0cb6/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/FileSystem/branch/dev)  |
+| Travis  | [![Travis](https://travis-ci.org/aspnet/FileSystem.svg?branch=dev)](https://travis-ci.org/aspnet/FileSystem)  |
 
 File Provider abstractions. Contains file system abstractions and file system globbing.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+In addition, there are some other FileProvider implementations that you may find useful:
```
Comment (Eilon) [inline README.md]: Slightly tweaked words: `In addition, there are some **community** FileProvider implementations that you may find useful:`

**Comment by dazinator** [comment]
Think this just needs someone to approve the changes

**Comment by Eilon** [comment]
@dazinator if you update per the suggestion I made, we can get it merged ASAP. Thanks!

**Comment by dazinator** [comment]
Is that better!?

**Comment by Eilon** [comment]
Sorry, I didn't mean to include the `**` in there, that was just to emphasize the change I made ☹️

**Comment by dazinator** [comment]
In the words of a certain famous cartoon character: doh!

**Comment by Eilon** [comment]
@dazinator if you remove the `**` from there I can merge it in ASAP. Sorry for all the trouble!

**Comment by dazinator** [comment]
Ok - how's that? :-)

**Comment by Eilon** [comment]
@dazinator thank you!

**Pull request merged**